wren-ui/src/apollo/server/adaptors/ibisAdaptor.ts

```diff
@@ -64,6 +75,7 @@ const dataSourceUrlMap: Record<SupportedDataSource, string> = {
   [SupportedDataSource.MYSQL]: 'mysql',
   [SupportedDataSource.MSSQL]: 'mssql',
   [SupportedDataSource.CLICK_HOUSE]: 'clickhouse',
+  [SupportedDataSource.TRINO]: 'trino',
```

Comment:
Can you add some test cases in the ibisAdaptor.test.ts file for the Trino data source